src/VecSim/algorithms/svs/svs_serializer.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+ */
+
+#pragma once
+
+#include "svs/index/vamana/dynamic_index.h"
+#include "svs/index/vamana/multi.h"
```

Comment:
It is not safe to put such includes inside the body of `SVSIndex` class definition.